common/src/main/java/com/wynntils/framework/feature/Feature.java

```diff
@@ -9,12 +9,14 @@
 public abstract class Feature {
     /** Called on a feature's activation */
     public void onEnable() {
-        WynntilsMod.eventBus.register(this);
+        WynntilsMod.EVENT_BUS.register(this);
+        WynntilsMod.EVENT_BUS.register(this.getClass());
```

Comment:
Is there a case where both static and nonstatic are used?

Reply:
Right now, no. (It was before I rebased it on your latest changes :))

Reply:
But if the registration should be done by the abstract superclass, I think we should cater to both static and instance listeners.